eng/tools/TypeSpecValidation/src/rules/format.ts

```diff
@@ -8,9 +8,9 @@ export class FormatRule implements Rule {
 
   async execute(folder: string): Promise<RuleResult> {
     // Format parent folder to include shared files
-    let [stdOutput, errorOutput] = await runCmd(`npx tsp format ../**/*.tsp`, folder);
 
-    let success = errorOutput == null ? false : true;
+    let [err, stdOutput, errorOutput] = await runCmd(`npx tsp format ../**/*.tsp`, folder);
+    let success = err || errorOutput ? false : true;
```

Comment:
```suggestion
    // Failing on both err and errorOutput because of known bug in tsp format
    // https://github.com/microsoft/typespec/issues/2323
    let success = err || errorOutput ? false : true;
```